src/keys-api/keys-api.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable, LoggerService, Inject } from '@nestjs/common';
+import { FetchService } from '@lido-nestjs/fetch';
+import { AbortController } from 'node-abort-controller';
+import { FETCH_REQUEST_TIMEOUT } from './keys-api.constants';
+import { SRModuleKeysResponse, SRModuleListResponse } from './interfaces';
+import { WINSTON_MODULE_NEST_PROVIDER } from 'nest-winston';
+import { Configuration } from 'common/config';
+
+@Injectable()
+export class KeysApiService {
```

Comment:
I think we should collect the Keys API version and send it to the message broker

Reply:
And it's worth collecting chainId from Keys API and checking it against the chainId of the council daemon EL API providers